api/lib/domain/usecases/authenticate-user.js

```diff
@@ -20,18 +21,38 @@ function _checkUserAccessScope(scope, user) {
   }
 }
 
+async function _checkScoUserAccessCertifScope({ scope, user, certificationCenterMembershipRepository }) {
+  const userBelongsToSco = await _doesUserBelongsToScoCenter({ user, certificationCenterMembershipRepository });
+  const blockingAccessToScoUserFeatureToggleIsOn = config.featureToggles.certifBlockingScoUserAccess;
+  const scopeIsPixCertif = scope === apps.PIX_CERTIF.SCOPE;
+
+  if (scopeIsPixCertif &&
+        blockingAccessToScoUserFeatureToggleIsOn &&
```

Comment:
mettres les is en premier : 
isScopePixCertif
isBlockingAccessToScoUserFeatureToggleEnabled

et does en premier : 
doesUserBelongToScoCertificationCenter

Reply:
Utiliser le toggle de prescription sco : 
```javascript
const shouldBlockScoUser = !config.featureToggles.certifPrescriptionSco // à vérifier
if (scopeIsPixCertif &&
        shouldBlockScoUser &&
        userBelongsToSco)
{
    throw new ForbiddenAccess(apps.PIX_CERTIF.USER_SCO_BLOCKED_CERTIFICATION_MSG);
}
```